### PR TITLE
Fix bugzilla minor issues

### PR DESCRIPTION
**Pull request by khalifapro**

### CODE REVIEW

File: bugzilla/enricher.go [hunk 1]
```diff
@@ -273,3 +264,26 @@ func (e *Enricher) EnrichItem(rawItem BugRaw, now time.Time) (*BugEnrich, error)
...
 }
+
+func getCont(con *Person) (string, string) {
+	key := "username"
+	val := ""
+
+	if con.Name != "" {
+		val = con.Name
+		key = "name"
+		if strings.Contains(con.Name, "@") {
```
Comment (1010sachin) [inline bugzilla/enricher.go:275]: What if the name is 'XYZ@ABC' will this still be considered email?

File: bugzilla/enricher.go [hunk 2]
```diff
@@ -273,3 +264,26 @@ func (e *Enricher) EnrichItem(rawItem BugRaw, now time.Time) (*BugEnrich, error)
...
+			key = "email"
+		}
+		return key, val
+	}
+
+	if con.Username != "" {
+		val = con.Username
+		if strings.Contains(con.Username, "@") {
+			key = "email"
```
Comment (1010sachin) [inline bugzilla/enricher.go:284]: Same as above

File: bugzilla/enricher.go (hunk 1 above)
Comment (khalifapro) [reply bugzilla/enricher.go:275]: Yes

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

File: bugzilla/enricher.go (hunk 1 above)
Comment (khalifapro) [reply bugzilla/enricher.go]: @1010sachin the main issue with bugzilla was viewing email address which made a privacy problem so they view name instead, and we have records with email and others with name. added Isemailvalid functionality.

**Pull request merged**